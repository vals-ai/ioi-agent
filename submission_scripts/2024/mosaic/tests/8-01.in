version https://git-lfs.github.com/spec/v1
oid sha256:86ce15aa956c28558840fedd8c5c8bd2b91762f45e2f3e3452f1eb5d3fcceaf4
size 5956000
