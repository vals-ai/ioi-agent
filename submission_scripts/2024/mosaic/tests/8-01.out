version https://git-lfs.github.com/spec/v1
oid sha256:ab934157b3fd2cdbabe6d053cd3e2802e9459fa44609f20eab83ba1975d83341
size 2088485
